snapd.code-workspace

```diff
@@ -0,0 +1,52 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"go.testEnvVars": {"LANG": "C.UTF-8"},
+		"go.testFlags": ["-v","-check.v"]
+	},
+	"extensions": {
+		"recommendations": [
+			"golang.go", 
+			"766b.go-outliner", 
+			"jerrygoyal.shortcut-menu-bar",
+			"eamodio.gitlens", 
+			"aaron-bond.better-comments",
+			"vscodevim.vim",
+			"tuttieee.emacs-mcx",
```

Comment:
This one is also something I would not put into the workspace. All the users who want vim or emacs behavior in vscode already have that enabled.

Reply:
I don't think that applies. People who don't use vscode probably don't want vim and emacs modes installed as well. Things like this belong in a tutorial or in personal preference file.

Reply:
@zyga I see your point. I thought actually the other way around. I thought that vim emacs users my want to use VSCODE just for debugging and I wanted to make them feel home. But perhaps I am trying to much. "The only enemy of good is perfect."